assets/src/editor/events/dates-and-times/editor-date/filter-bar/with-date-entities-list-filter-state.js

```diff
@@ -59,6 +62,14 @@ export default createHigherOrderComponent(
 					'datesView',
 					datesView
 				),
+				filteredDateEntities: getEntitiesByIds(
+					'datetime',
+					store.getFilter(
+						'event-editor-dates-list',
+						'filteredDateIds',
+						DEFAULT_EMPTY_ARRAY
+					)
+				),
```

Comment:
I was able to use the existing structure of the `eventespresso/filter-state` store to track the filtered entities.  Note this value retrieves the actual entities matching the stored ids from the state.